tests/messages/any/get.test.tsx

```diff
@@ -0,0 +1,197 @@
+import { any } from "../../index";
+import { BaseMessage, Chain, MessageType, StoreMessage } from "../../../src/messages/message";
+
+describe("Test features from GetUnique", () => {
+    it("Try by Hash with type-guard resolve", async () => {
+        const res = await any.GetUnique({
+            hash: "87e1e2ee2cbe88fa2923042b84b2f9c69410005ca7dd40193838bf9bad18e12c",
+        });
+
+        expect(any.is.Store(res)).toStrictEqual(true);
+        if (any.is.Store(res))
+            expect(res.content.item_hash).toStrictEqual("QmZyVbZm6Ffs9syXs8pycGbWiTa9yiGoX1b9FSFpTjaixK");
+    });
+
+    it("Try by Hash with templating resolve", async () => {
+        const res = await any.GetUnique<StoreMessage>({
+            hash: "87e1e2ee2cbe88fa2923042b84b2f9c69410005ca7dd40193838bf9bad18e12c",
+        });
+
+        expect(res.content.item_hash).toStrictEqual("QmZyVbZm6Ffs9syXs8pycGbWiTa9yiGoX1b9FSFpTjaixK");
+    });
+
+    it("If the message can't be resolve, it should failed", async () => {
+        await expect(
+            any.GetUnique({
+                hash: "w87e1e2ee2cbe88fa2923042b84b2f9c694w10005ca7dd40193838bf9bad18e12cw",
+            }),
+        ).rejects.toThrow("No messages found for: w87e1e2ee2cbe88fa2923042b84b2f9c694w10005ca7dd40193838bf9bad18e12cw");
+    });
+});
```

Comment:
What is this message hash and why would it automatically fail ?

Reply:
I randomly added chars to an already existing hash to invalidate it
Another approach with a string like `wontexist` might be safer, but it seems ok for me like this